src/_premake_main.lua

```diff
@@ -387,6 +443,7 @@
 		m.bake,
 		m.postBake,
 		m.validate,
+		m.nonActionGeneration,
```

Comment:
Don't hesitate if you have better naming.

Reply:
Maybe git hook installation?

Reply:
Changed